cmd/workspace/lakeview/lakeview.go

```diff
@@ -70,35 +70,33 @@ func newCreate() *cobra.Command {
 	cmd := &cobra.Command{}
 
 	var createReq dashboards.CreateDashboardRequest
+	createReq.Dashboard = &dashboards.Dashboard{}
 	var createJson flags.JsonFlag
 
 	// TODO: short flags
 	cmd.Flags().Var(&createJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
-	cmd.Flags().StringVar(&createReq.ParentPath, "parent-path", createReq.ParentPath, `The workspace path of the folder containing the dashboard.`)
-	cmd.Flags().StringVar(&createReq.SerializedDashboard, "serialized-dashboard", createReq.SerializedDashboard, `The contents of the dashboard in serialized string form.`)
-	cmd.Flags().StringVar(&createReq.WarehouseId, "warehouse-id", createReq.WarehouseId, `The warehouse ID used to run the dashboard.`)
-
-	cmd.Use = "create DISPLAY_NAME"
+	cmd.Flags().StringVar(&createReq.Dashboard.CreateTime, "create-time", createReq.Dashboard.CreateTime, `The timestamp of when the dashboard was created.`)
+	cmd.Flags().StringVar(&createReq.Dashboard.DashboardId, "dashboard-id", createReq.Dashboard.DashboardId, `UUID identifying the dashboard.`)
+	cmd.Flags().StringVar(&createReq.Dashboard.DisplayName, "display-name", createReq.Dashboard.DisplayName, `The display name of the dashboard.`)
+	cmd.Flags().StringVar(&createReq.Dashboard.Etag, "etag", createReq.Dashboard.Etag, `The etag for the dashboard.`)
+	cmd.Flags().Var(&createReq.Dashboard.LifecycleState, "lifecycle-state", `The state of the dashboard resource. Supported values: [ACTIVE, TRASHED]`)
+	cmd.Flags().StringVar(&createReq.Dashboard.ParentPath, "parent-path", createReq.Dashboard.ParentPath, `The workspace path of the folder containing the dashboard.`)
+	cmd.Flags().StringVar(&createReq.Dashboard.Path, "path", createReq.Dashboard.Path, `The workspace path of the dashboard asset, including the file name.`)
+	cmd.Flags().StringVar(&createReq.Dashboard.SerializedDashboard, "serialized-dashboard", createReq.Dashboard.SerializedDashboard, `The contents of the dashboard in serialized string form.`)
+	cmd.Flags().StringVar(&createReq.Dashboard.UpdateTime, "update-time", createReq.Dashboard.UpdateTime, `The timestamp of when the dashboard was last updated by the user.`)
+	cmd.Flags().StringVar(&createReq.Dashboard.WarehouseId, "warehouse-id", createReq.Dashboard.WarehouseId, `The warehouse ID used to run the dashboard.`)
+
+	cmd.Use = "create"
 	cmd.Short = `Create dashboard.`
 	cmd.Long = `Create dashboard.
   
-  Create a draft dashboard.
-
-  Arguments:
-    DISPLAY_NAME: The display name of the dashboard.`
```

Comment:
This seems to be caused by display_name not marked as required in the OpenAPI https://github.com/databricks/databricks-sdk-go/blame/776b63cf7f8f155f7642b0320d2bea0ced011914/service/dashboards/model.go#L54

Reply:
cc @renaudhartert-db